test/api-digester/Outputs/stability-stdlib-abi.swift.expected

```diff
@@ -604,3 +604,5 @@ Var String.Index._utf16Index has been removed (deprecated)
 
 Func MutableCollection.withContiguousMutableStorageIfAvailable(_:) has been added as a protocol requirement
 Func Sequence.withContiguousStorageIfAvailable(_:) has been added as a protocol requirement
+
+Class AnyKeyPath is now with @objc
```

Comment:
Oh wait, this is a problem after all, because we don't want people using it in `@objc` methods. :-(

Reply:
Hm. We have an implementation of something like that for classes with generic ancestry called ObjCRuntimeNameAttr, but you can't actually write that in source right now. We might need a compiler change here.

Reply:
Simple changes are never simple, huh. Thanks for the pointer, I’ll look into that.

Reply:
I just pushed an update to add `@_objcRuntimeName` and use it for this. Hopefully I didn't embarrass myself too badly with the parser modifications. The result is, I believe, what we want now.